docs/api.md

```diff
@@ -1926,7 +1926,272 @@ HTTP/1.1 200 Success
 
 ## AdministrativeLevel
 
-> TODO
+`AdministrativeLevel` Subdivisions of areas/territories/jurisdictions recognized by governments or other organizations for administrative purposes.
+
+### AdministrativeLevel Schema
+
+`AdministrativeLevel` have the following attributes:
+
+<br />
+
+|    Name     |  Type  |                                                      Description                                                       |
+| :---------: | :----: | :--------------------------------------------------------------------------------------------------------------------: |
+|    \_id     | String |                               Unique universal identifier of this administrative level.                                |
+|    name     | Object |                             Human-translatable-readable name for the administrative level.                             |
+| description | Object |                         Human-translatable-readable description for the administrative level.                          |
+|    color    | String |               A color code(in hexadecimal format) used to differentiate administrative levels visually.                |
+|    level    | Number |                            Human-translatable-readable level for the administrative level.                             |
```

Comment:
This is not supposed to be human translatable because it is just a number, it doesn't support multilanguage